composer.json

```
@@ -6,20 +6,19 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.7",
-        "php": ">=7.2"
+        "drupal/core": "^8.9 || ^9.1",
+        "php": ">=7.3"
     },
     "require-dev": {
-        "composer/installers": "~1.5",
+        "composer/installers": "^1.11",
```

We can keep `~` for this constraint
```suggestion
        "composer/installers": "~1.11",
```